packages/plugin-mparticle/lib/index.ts

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable no-unused-vars, class-methods-use-this, import/no-unresolved */
+/* eslint-disable no-underscore-dangle */
+import {
+  Options, Event, Properties, PluginBase,
+} from '@itly/sdk';
+import Mparticle from '@itly/mparticle-web-sdk';
+
+export type MparticleOptions = {
+  itly?: any,
+  isDevelopmentMode?: boolean,
+};
+
+export default class MparticleBrowserPlugin extends PluginBase {
+  static ID: string = 'mparticle';
+
+  private $itly = 'audit';
+
+  private mparticle?: any;
+
+  constructor(
+    private apiKey: string,
+    private options: MparticleOptions = { },
+  ) {
+    super();
+  }
+
+  id = () => MparticleBrowserPlugin.ID;
+
+  load() {
+    this.mparticle = Mparticle.getInstance();
+    if (!this.mparticle?._Store) {
```

Comment:
Scary to have to reach into private internals to tell whether we're initialized or not (and on other platforms where private means private, might not even possible). There is just no public API we might be able to infer this out of instead?